tripy/nvtripy/frontend/module/instancenorm.py

```diff
@@ -0,0 +1,158 @@
+#
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from dataclasses import dataclass
+
+from nvtripy import constants, export, utils
+from nvtripy.common import datatype
+from nvtripy.common.exception import raise_error
+from nvtripy.frontend.module.module import Module
+from nvtripy.frontend.module.parameter import DefaultParameter
+from nvtripy.frontend.tensor import Tensor
+
+from nvtripy.frontend.ops import utils as op_utils
+from nvtripy.utils import wrappers
+from nvtripy.trace.ops.instancenorm import InstanceNorm as InstanceNormOp
+
+
+@wrappers.interface(
+    dtype_constraints={"input": "T1", "weight": "T1", "bias": "T1", wrappers.RETURN_VALUE: "T1"},
+    dtype_variables={"T1": ["float32", "float16", "bfloat16"]},
+)
+def instancenorm(
+    input: "nvtripy.Tensor",
+    weight: "nvtripy.Tensor",
+    bias: "nvtripy.Tensor",
+    num_channels: int,
+    eps: float = 1e-5,
```

Comment:
Same rationale as for the trace op:
```suggestion
    eps: float,
```